processing/data_collection/gazette/spiders/ap_macapa.py

```diff
@@ -0,0 +1,79 @@
+import datetime as dt
+
+import dateparser
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class ApMacapaSpider(BaseGazetteSpider):
+
+    name = "ap_macapa"
+    allowed_domains = ["macapa.ap.gov.br"]
+    start_date = None
+
+    TERRITORY_ID = "1600303"
+
+    def __init__(self, start_date=None, end_date=None, *args, **kwargs):
+        if start_date is None:
+            # Start from yesterday
+            start_date = dt.date.today() + dt.timedelta(days=-1)
+        else:
+            start_date = dateparser.parse(start_date)
+
+        if end_date is None:
+            # End today
+            end_date = dt.date.today()
+        else:
+            end_date = dateparser.parse(end_date)
+
+        self.start_date = start_date
+        self.end_date = end_date
+
+    def start_requests(self):
+        date_format = "%d/%m/%Y"
+        base_url = "https://macapa.ap.gov.br/"
+
+        target_date = self.start_date
+        data = {
+            "s": "",
+            "post_type": "official_diaries",
+            "search": "official_diaries",
+            "official_diary_number": "",
+        }
+        while target_date <= self.end_date:
+            formatted_date = target_date.strftime(date_format)
```

Comment:
It seems that `date_format` is only used here, so I'd use the string here and not define a new variable.
It would be easier to read, I think

Reply:
Done.